gems/sorbet/bin/srb

```diff
@@ -60,6 +60,11 @@ typecheck() {
   if [ -n "$SRB_SORBET_EXE" ]; then
     # shellcheck disable=SC2086
     "$SRB_SORBET_EXE" "${args[@]}"
+  elif command -v sorbet > /dev/null; then
```

Comment:
You can already simulate what you'd like to do by running

```bash
SRB_SORBET_EXE=sorbet srb tc
```

is that good enough?

We've historically **not** added the change implemented here because it could lead to surprising / hard to diagnose behavior (people asking "why am i getting weird errors" and it's because there's a `sorbet` executable lurking on the path somewhere.

Reply:
Well I am about to submit a `homebrew` formulae that will install sorbet.

I find the ability to find it automatically on the path much simpler than having to set `SRB_SORBET_EXE`

In fact for installation instructions -- I would make it clear to run `brew install sorbet` as that works in all platform cases: MRI & JRuby

We can add an echo statement declaring the path at which it is running sorbet ?